gpiozero/boards.py

```diff
@@ -1520,7 +1520,7 @@ class TrafficHat(CompositeOutputDevice):
         See :doc:`api_pins` for more information (this is an advanced feature
         which most users can ignore).
 
-    .. _Ryanteck Traffic HAT: https://ryanteck.uk/hats/1-traffichat-0635648607122.html
+    .. _Ryanteck Traffic HAT: https://uk.pi-supply.com/products/traffic-hat-for-raspberry-pi
```

Comment:
I guess the `_Ryanteck Traffic HAT` here should be `_Pi Supply Traffic HAT` in order to match the earlier link?

Reply:
I'll double check but that sounds right to me. I'll get all these fixes done today.

Reply:
That naming is now fixed.